qemu/tests/bridge_qinq.py

```diff
@@ -288,8 +288,8 @@ def compare_host_guest_md5sum(name):
 
         # scp file to guest with L2 vlan tag
         file_size = int(params.get("file_size", "4096"))
-        host_path = params.get("host_path", "/tmp/transferred_file")
-        guest_path = params.get("guest_path", "/tmp/transferred_file")
+        host_path = os.path.join(data_dir.get_tmp_dir(), "transferred_file")
```

Comment:
Can use `test.tmpdir`.

Reply:
Updated, please help me review it again, thanks a lot.